Source/Scene/Cesium3DTileStyle.js

```diff
@@ -209,7 +209,7 @@ define([
         },
 
         /**
-         * Gets or sets the {@link StyleExpression} object used to evaluate the style's <code>show</code> property.
+         * Gets or sets the {@link StyleExpression} object used to evaluate the style's <code>show</code> property. Alternative an object defining a show style can be used.
```

Comment:
Also tweak the new description a bit: `Alternatively a boolean, string, or object defining a show style can be used.`